example/patches/react-native+0.71.14.patch

```diff
@@ -0,0 +1,194 @@
+diff --git a/node_modules/react-native/sdks/hermes/hermes-engine.podspec b/node_modules/react-native/sdks/hermes/hermes-engine.podspec
```

Comment:
See issue here on incorrect checksum error for boost dependency. Only affects our RN example app - https://github.com/facebook/react-native/issues/42180